tools/test-proxy/documentation/asset-sync/README.md

```diff
@@ -65,6 +65,10 @@ Interactions with the external assets repository are accessible when the proxy i
 
 Each of these CLI Commands takes an `assets.json` argument that provides the _context_ that should be pushed to the external repository.
 
+### A note about using on Windows + WSL
+
+When using a Windows machine, it is technically possible to invoke tests from WSL against a windows clone. That path would appear under `/mnt/c/path/to/your/repo`. This is _not_ a supported scenario, as the `test-proxy` shells out to git for the push/restore actions. Running a `git restore` from _linux_ against a repo that was cloned down using a _windows_ git client can have unexpected results. Better to avoid the situation entirely and use an entirely separate clone for work on WSL.
```

Comment:
```suggestion
When using a Windows machine, it is technically possible to invoke tests from WSL against a windows clone. That path would appear under `/mnt/c/path/to/your/repo`. This is _not_ a supported scenario, as the `test-proxy` shells out to git for the push/restore actions. Running a `git push/pull` from _linux_ against a repo that was cloned down using a _windows_ git client can have unexpected results. Better to avoid the situation entirely and use an entirely separate clone for work on WSL.
```